pkg/cli/cli.go

```diff
@@ -23,14 +24,14 @@ var (
 // CLI exposes methods for configuring clients that execute greymatter CLI commands.
 type CLI struct {
 	*sync.RWMutex
-	clients         map[string]*client
-	meshconfigsTmpl cue.Value
-	mTLSEnabled     bool
+	client      *Client
+	operatorCUE cuemodule.OperatorCUE
```

Comment:
```suggestion
	operatorCUE *cuemodule.OperatorCUE
```
This feels like it should be a pointer. Passing a copied value could trip us up.